pyaedt/generic/filesystem.py

```diff
@@ -73,17 +73,23 @@ def remove(self):
 
     def copyfile(self, src_file, dst_filename=None):
         """
+        Copy a file to the scratch directory. The target filename is optional.
+        If omitted, the target file name is identical to the source file name.
 
         Parameters
         ----------
         src_file : str
             Source File with fullpath
         dst_filename : str, optional
-            Optional destination filename with extensione
+            Destination filename with the extension. The default is ``None``,
+            in which case the destination file is given the same name as the
+            source file.
 
 
         Returns
         -------
+        dst_file : str
+            Full path and file name of the copied file.
```

Comment:
```suggestion
            Full path and filename of the copied file.
```
I personally hate "filename" as one word but that is the style guideline.